sympy/assumptions/ask.py

```diff
@@ -129,6 +131,9 @@ def ask(proposition, assumptions=True, context=global_assumptions):
     else:
         key, expr = Q.is_true, sympify(proposition)
 
+    if isinstance(proposition, (bool, BooleanAtom)):
+        return sympify(proposition)
```

Comment:
Why `BooleanTrue` instead of `bool`?

This makes the output less consistent. In both branches, master and this one, `ask` returns `bool`.

This change is making `ask(True) is True` `False`.